src/api/users/vendorRoutes.js

```diff
@@ -3,10 +3,5 @@ import { auth } from '../../middleware/auth';
 import {createVendor} from "../../controllers/users/vendorFunctions" 
 
 const router =Express.Router()
```

Comment:
Make sure you are using same code formatter like the rest of the team because you have many empty lines